docs/de/docs/index.md

```diff
@@ -293,175 +287,175 @@ def update_item(item_id: int, item: Item):
+  * Da der `q`-Parameter mit `= None` deklariert ist, ist er optional.
+  * Ohne das `None` wäre er erforderlich (wie der Body im Fall von `PUT`).
+* Bei `PUT`-Anfragen an `/items/{item_id}` wird der Body als JSON gelesen:
+  * Prüfen Sie, ob er ein erforderliches Attribut `name` hat, das ein `str` sein sollte.
+  * Prüfen Sie, ob er ein erforderliches Attribut `price` hat, das ein `float` sein muss.
+  * Prüfen Sie, ob es ein optionales Attribut `is_offer` hat, das ein `bool` sein sollte, falls vorhanden.
+  * All dies würde auch für tief verschachtelte JSON-Objekte funktionieren.
+* Konvertiert automatisch von und nach JSON.
+* Alles mit OpenAPI dokumentieren, das kann von:
+  * Interaktive Dokumentationssysteme.
+  * Automatische Client-Code-Generierung, für viele Sprachen.
+* 2 interaktive Dokumentation-Webschnittstellen direkt zur Verfügung stellen.
 
 ---
 
-We just scratched the surface, but you already get the idea of how it all works.
+Wir haben nur an der Oberfläche gekratzt, aber Sie bekommen schon eine Vorstellung davon, wie das Ganze funktioniert.
 
-Try changing the line with:
+Versuchen Sie, diese Zeile zu ändern mit:
 
 ```Python
     return {"item_name": item.name, "item_id": item_id}
 ```
 
-...from:
+...von:
 
 ```Python
         ... "item_name": item.name ...
 ```
 
-...to:
+...zu:
 
 ```Python
         ... "item_price": item.price ...
 ```
 
-...and see how your editor will auto-complete the attributes and know their types:
+...und sehen Sie, wie Ihr Editor die Attribute automatisch ausfüllt und ihre Typen kennt:
 
 ![editor support](https://fastapi.tiangolo.com/img/vscode-completion.png)
 
-For a more complete example including more features, see the <a href="https://fastapi.tiangolo.com/tutorial/">Tutorial - User Guide</a>.
+Ein vollständigeres Beispiel mit weiteren Funktionen finden Sie im <a href="https://fastapi.tiangolo.com/tutorial/">Tutorial - Benutzerhandbuch</a>.
 
-**Spoiler alert**: the tutorial - user guide includes:
+**Spoiler-Alarm**: Das Tutorial - Benutzerhandbuch enthält:
 
-* Declaration of **parameters** from other different places as: **headers**, **cookies**, **form fields** and **files**.
-* How to set **validation constraints** as `maximum_length` or `regex`.
-* A very powerful and easy to use **<abbr title="also known as components, resources, providers, services, injectables">Dependency Injection</abbr>** system.
-* Security and authentication, including support for **OAuth2** with **JWT tokens** and **HTTP Basic** auth.
-* More advanced (but equally easy) techniques for declaring **deeply nested JSON models** (thanks to Pydantic).
-* Many extra features (thanks to Starlette) as:
-    * **WebSockets**
-    * **GraphQL**
-    * extremely easy tests based on `requests` and `pytest`
-    * **CORS**
-    * **Cookie Sessions**
-    * ...and more.
+* Deklaration von **Parametern** von anderen verschiedenen Stellen wie: **Kopfzeilen**, **Cookies**, **Formularfelder** und **Dateien**.
```

Comment:
"Kopfzeilen" could be ambiguous. Even in german I would stick to the english `Headers`

Reply:
"Header-Felder" würde ich nehmen, dann ist auch [das RFC und die Wikipedia](https://de.m.wikipedia.org/wiki/Liste_der_HTTP-Headerfelder) glücklich. Hab ich auch so in meinem unten erwähnten Edit verwendet.